pkg/kn/commands/service/configuration_edit_flags.go

```diff
@@ -103,11 +121,15 @@ func (p *ConfigurationEditFlags) addSharedFlags(command *cobra.Command) {
 			"Accepts golang templates, allowing {{.Service}} for the service name, "+
 			"{{.Generation}} for the generation, and {{.Random [n]}} for n random consonants.")
 	p.markFlagMakesRevision("revision-name")
+	command.Flags().StringVar(&p.ServiceAccountName, "service-account-name", "-", "Service account name. To unset, specify \"-\".")
+	p.markFlagMakesRevision("service-account-name")
+	command.Flags().StringVar(&p.ImagePullPolicy, "image-pull-policy", "-", "Image pull policy (e.g., always, if-not-present, or never). To unset, specify \"-\".")
```

Comment:
`image-pull-policy` is not supported by Knative serving by design — revisions are always constant images by hash.

Reply:
Oh, I understood. I will remove the option.